beets/autotag/mb.py

```diff
@@ -237,6 +246,11 @@ def track_info(recording, index=None, medium=None, medium_index=None,
                 arranger.append(artist_relation['artist']['name'])
     if arranger:
         info.arranger = u', '.join(arranger)
+    if work:
+        info.work = u', '.join(work)
+        info.work_id = u', '.join(work_id)
+        if all(dis for dis in work_disambig):
```

Comment:
Under what circumstances would `dis` be false in this list? Would it be better to avoid inserting it into the list in the first place?

Reply:
Sorry, it was me not understanding correctly what ```join``` does.

Reply:
I think it was meant to not add an empty ```work_disambig``` field.